src/pudl/package_data/settings/etl_full.yml

```diff
@@ -3,159 +3,159 @@
 # Settings for ferc_to_sqlite script
 ###########################################################################
 ferc_to_sqlite_settings:
-  ferc1_dbf_to_sqlite_settings:
```

Comment:
I'm guessing you didn't mean to commit these changes to the ETL settings?

Reply:
Ooopsies thanks for the catch. Just reverted the change.